assets/src/edit-story/components/panels/textStyle/useRichTextFormatting.js

```diff
@@ -43,6 +43,11 @@ function isEqual(a, b) {
   // Note: `null` is a falsy object, that would cause an error if first
   // check is removed.
   const isPattern = a && typeof a === 'object' && (a.type || a.color);
+
+  if (isPattern && (a === MULTIPLE_VALUE || b === MULTIPLE_VALUE)) {
+    return false;
+  }
```

Comment:
I think this is basically it, but should it be `isPatternA` and `isPatternB` (both with the same respective checks that is currently only for `a`) and only if both `a` and `b` are patterns then try `isPatternEqual`?  It just seems that all the checks were for a while ignoring that b needs to be checked too.

Reply:
If either are true then

```
const p1CSS = convertToCSS(generatePatternStyles(p1, patternType));
const p2CSS = convertToCSS(generatePatternStyles(p2, patternType));
```

will get called by `isPatternEqual(a, b)`

which will cause a crash since the string `(( Multiple ))` cannot be destructured to `{ r, g, b, a }`. So I need it to short circuit if _either_ are not a valid object.

Reply:
How will `isPatternEqual` get called if you do this?

```js
const isPatternA = a && typeof a === 'object' && (a.type || a.color);
const isPatternB = b && typeof b === 'object' && (b.type || b.color);

return (isPatternA && isPatternB) ? isPatternEqual(a, b) : a === b;
```

Reply:
Maybe even like this?

```js
const isPatternA = a?.type || a?.color;
const isPatternB = b?.type || b?.color;

return (isPatternA && isPatternB) ? isPatternEqual(a, b) : a === b;
```

Reply:
Yeah that could work.

Reply:
Would be great if you could give it a try 👍  'cause I think that's what Mariano was suggesting :)